test/solve_by_elim.lean

```diff
@@ -99,7 +99,7 @@ end
 -- Verifying that `solve_by_elim*` backtracks when given multiple goals.
 example (n m : ℕ) (f : ℕ → ℕ → Prop) (h : f n m) : ∃ p : ℕ × ℕ, f p.1 p.2 :=
 begin
-  repeat { split },
+  repeat { fsplit },
```

Comment:
Was there a reason for this change?

Reply:
It wasn't actually testing what the comment said it was testing!

With `split`, the first goal was `f ?m_1 ?m_2`, which you can immediately solve with `h`, and the other goals disappear by unification.

With `fsplit`, it first tries solving both metavariables with `n`, then can't solve `f n n`, so backtracks to try `m` on the second metavariable, then succeeds with `h`.